subscriber/src/subscriber/subscriber.bal

```diff
@@ -41,23 +25,28 @@ public function main (string? username = (),        // my username
                       boolean 'json = false,        // do I want json?
                       boolean 'xml = false,         // do I want xml?
                       boolean image = false,        // do I want the image?
-                    //   boolean html = false,         // do I want HTML?
-                    //   boolean sorted = true,        // do I want HTML results sorted highest to lowest
+                      boolean html = false,         // do I want HTML?
```

Comment:
Need to add this option to info.txt too.

Reply:
Added via dcbba294287a137a375d078d8d6e514abaa3ad3f